ApiBundle/Transformer/ContentCollectionTransformer.php

```diff
@@ -23,7 +24,9 @@ public function transform($contentCollection, $contentType = null)
         $facade = new ContentCollectionFacade();
 
         foreach ($contentCollection as $content) {
-            $facade->addContent($this->getTransformer('content')->transform($content));
+            if ($this->authorizationChecker->isGranted(ContentTypeForContentPanelStrategy::ROLE_ACCESS_CONTENT_TYPE_FOR_CONTENT, $content)) {
```

Comment:
it's not better to just hide the button to edit content than not display the content himself.
Same for the node